src/ARToolkit/ARSceneView.iOS.cs

```diff
@@ -141,37 +141,41 @@ private void OnCameraDidChangeTrackingState(ARSession session, ARCamera camera)
 
         private void OnWillRenderScene(ISCNSceneRenderer renderer, SCNScene scene, double timeInSeconds)
         {
-            if (_tracking && ARSCNView != null)
+            CoreFoundation.DispatchQueue.MainQueue.DispatchAsync(() =>
```

Comment:
I don't have context why this change is now required. This shouldn't be necessary. Commit comment is "Fix crash AR sceneview ios" with no reference to issue.